rest-api-with-express/package.json

```diff
@@ -10,10 +10,10 @@
     "build": "exit 0"
   },
   "dependencies": {
-    "@romcal/calendar.france": "3.0.0-dev.67",
-    "@romcal/calendar.general-roman": "3.0.0-dev.67",
-    "express": "^4.18.2",
-    "romcal": "3.0.0-dev.67"
+    "@romcal/calendar.france": "3.0.0-dev.77",
+    "@romcal/calendar.general-roman": "3.0.0-dev.77",
+    "express": "^4.19.2",
```

Comment:
You have also update the Express version. It might be a good idea to say so in the PR description.